packages/office-addin-debugging/src/commands.ts

```diff
@@ -95,7 +77,9 @@ export async function start(manifestPath: string, appType: string | undefined, c
 
 export async function stop(manifestPath: string, appType: string | undefined, command: commander.Command) {
     try {
-        await stopDebugging(manifestPath);
+        const appTypeToDebug: AppType | undefined = parseAppType(appType || process.env.npm_package_config_app_type_to_debug || "desktop");
```

Comment:
Since there is common code in start() and stop(), how about moving it to a determineAppType() function?

Are you planning to extend AppType to cover the specific platforms (e.g. "win32", etc.)?